src/main/java/org/helpberkeley/memberdata/Main.java

```diff
@@ -1598,26 +1470,30 @@ private static void workRequests(ApiClient apiClient, String usersFile) throws I
-                doCompletedDailyOrders(apiClient, request, users);
-            } else if (topicId == Constants.TOPIC_REQUEST_ONE_KITCHEN_DRIVER_MESSAGES.getId()) {
-                doOneKitchenDriverMessages(apiClient, request, users);
-            } else if (topicId == Constants.TOPIC_REQUEST_WORKFLOW.getId()) {
-                createWorkflowRequest(request);
-            } else if (topicId == Constants.TOPIC_POST_ONE_KITCHEN_RESTAURANT_TEMPLATE.getId()) {
-                doOneKitchenRestaurantTemplate(apiClient, request);
-            } else if (topicId == Constants.TOPIC_POST_RESTAURANT_TEMPLATE.getId()) {
-                doRestaurantTemplate(apiClient, request);
-            } else if (topicId == Constants.TOPIC_POST_COMPLETED_ONEKITCHEN_ORDERS.getId()) {
-                doCompletedOneKitchenOrders(apiClient, request, users);
-            } else if (topicId == Constants.TOPIC_REQUEST_DATA.getId()) {
-                doUpdateMemberData(apiClient, request, users);
-            } else {
-                assert topicId == Constants.TOPIC_REQUEST_DRIVER_ROUTES.getId() : topicId;
-                requestHandler.postStatus(WorkRequestHandler.RequestStatus.Failed,
-                        "Route requests not supported");
+            try {
+                if (topicId == Constants.TOPIC_REQUEST_DRIVER_MESSAGES.getId()) {
+                    doDriverMessages(apiClient, request, users);
+                } else if (topicId == Constants.TOPIC_POST_COMPLETED_DAILY_ORDERS.getId()) {
+                    doCompletedDailyOrders(apiClient, request, users);
+                } else if (topicId == Constants.TOPIC_REQUEST_ONE_KITCHEN_DRIVER_MESSAGES.getId()) {
+                    doOneKitchenDriverMessages(apiClient, request, users);
+                } else if (topicId == Constants.TOPIC_REQUEST_WORKFLOW.getId()) {
+                    createWorkflowRequest(request);
+                } else if (topicId == Constants.TOPIC_POST_ONE_KITCHEN_RESTAURANT_TEMPLATE.getId()) {
+                    doOneKitchenRestaurantTemplate(apiClient, request);
+                } else if (topicId == Constants.TOPIC_POST_RESTAURANT_TEMPLATE.getId()) {
+                    doRestaurantTemplate(apiClient, request);
+                } else if (topicId == Constants.TOPIC_POST_COMPLETED_ONEKITCHEN_ORDERS.getId()) {
+                    doCompletedOneKitchenOrders(apiClient, request, users);
+                } else if (topicId == Constants.TOPIC_REQUEST_DATA.getId()) {
+                    doUpdateMemberData(apiClient, request, users);
+                } else {
+                    assert topicId == Constants.TOPIC_REQUEST_DRIVER_ROUTES.getId() : topicId;
+                    requestHandler.postStatus(WorkRequestHandler.RequestStatus.Failed,
+                            "Route requests not supported");
+                }
+            } catch (MemberDataException ex) {
+                requestHandler.postStatus(WorkRequestHandler.RequestStatus.Failed, ex.getMessage());
```

Comment:
Go through the rest of Main and remove all of these meaningless checks after HttpClient.post():
```
       HttpResponse<?> response = apiClient.post(post.toJson());
       LOGGER.info("postConsumerRequests {}", response.statusCode() == HTTP_OK ? ...
```